cves/CVE-2014-0116.yml

```diff
@@ -205,4 +223,8 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
```

Comment:
How can vulnerabilities like this one be prevented in the future?

Reply:
I've added additional info about prevention to my PR itself in the latest commit.
In short, vulnerabilities like this one can be prevented by doing due diligence regarding researching pre-existing sanitization solutions (in this case, within the same repo), by communicating with team-members about such issues, and by creating more robust test cases.